docs/learn/specific-css.mdx

```diff
@@ -1,29 +1,27 @@
 ---
-slug: /specific-css
-title: Argos-specific CSS
+slug: /injecting-css
 sidebar_position: 3
 ---
 
-# Argos-specific CSS
+# Injecting CSS
 
-Understand the method for using Argos-focused CSS to exclude certain sections from appearing in your screenots.
+Argos integrations offer the possibility to inject custom CSS applied during the screenshot process to improve your screenshots or exclude some details.
 
-In addition to [Argos built-in helpers](https://argos-ci.com/docs/argos-helpers#helpers), Argos integrations offer the possibility to inject custom CSS applied during the screenshot process. It can be helpful for complex use cases that produce flaky visual diffs.
+## Usage
 
-## `argosCSS` SDK options
-
-All our SDKs supports an `argosCSS` options that allows you to specify custom CSS evaluated during the screenshot phase. The style will be removed from the page after the screenshot is taken.
+The `argosScreenshot()` command allows you to specify a `customCSS` option to inject CSS before capturing a screenshot. The style will be removed after the screenshot is taken.
 
 ```ts
-// Playwright / Puppeteer
-await argosScreenshot(page, "my-screenshot", {
-  argosCSS: `iframe { display: none }`,
-});
+const custom_style = `
+  iframe { display: none; }
+  background: { color: red; }
```

Comment:
This is invalid, please keep to the original example, it does not make sense to turn the bg in red. Also splitting in two lines makes it more complicated.

Reply:
ok

Reply:
By the way it was invalid, direct css without selector does not work.

Reply:
An example with two selectors is better I think

Reply:
Then do:

time { visibility: hidden; }
iframe { display: none; }

But I think it makes it less readable and more complicated, we got the idea, I mean CSS is a basic.